component/client/chat/ClientChat.tsx

```diff
@@ -48,15 +49,15 @@ class ClientChat extends React.Component<IClientChatProps, IClientChatState> {
       messages : []
     };
     room.setUrl(url, (data: IData, user) => {
-      this.setState((prevState, props) => {
+      this.setState((prevState: IClientChatState, props: IClientChatProps) => {
```

Comment:
Then you should fix it after this is merged. All this did was add types to the `setState` methods.